files/ohpc_install.yml

```diff
@@ -98,12 +106,28 @@
                                 enable_nfs_ohpc=True
                                 enable_nfs_home=True
                                 enable_helloworld=True
+                                enable_socket_direct=False
+                                mofed_install_opts=""
 
                         else
                                 echo 'Stateless not implemented'
                                 exit 1
                         fi
 
+                        if [ ${ib_install} == 'mofed' ]; then
+                            enable_opensm=False
+                            enable_ipoib=True
+                            enable_ifup=True
+                            enable_linux_ib=False
+                            enable_mellanox_ib=True
+                        elif [ ${ib_install} == 'foss' ]; then
+                            enable_opensm=False
+                            enable_ipoib=True
+                            enable_ifup=True
```

Comment:
The three above are identical, why are they inside the if/else?
If it's because it can be none, I'd default to "IB ON" put an *else* at the end disabling them.

Reply:
They are not identical (enable_mellanox_ib vs enable_linux_ib) but I bundled them all to be clearer.
The issue with default to 'IB On' is that it adds more crud to distinguish between QDF and the others to decide on MOFED/OSS.

Reply:
I didn't mean between themselves, I meant across both parts of the if/else blocks, meaning they can be commoned up to be clearer.

My proposal was:
1. Make their defaults what they are in the if/else (ie. opposite of what they are now)
2. Remove them from the if/else blocks
3. Add an "else" block at the end, with them in reverse